app/templates/application.hbs

```diff
@@ -2,45 +2,22 @@
   {{title (t "page-title")}}
   {{head-layout}}
   {{#if (await alert)}}
-    <div class="vl-alert vl-alert-relative vl-alert-relative--center
-        {{await type}}"
-         role="alert">
-      <div class="vl-alert__icon">
-        <i class="vl-vi vl-vi-alert-circle" aria-hidden="true"></i>
-      </div>
-      <button class="vl-alert__close" type="button">
-        <i class="vl-vi vl-vi-cross" aria-hidden="true"></i>
-        <span class="vl-u-visually-hidden">
-          {{t "close"}}
-        </span>
-      </button>
-      <div class="vl-alert__content">
-        <p class="vl-alert__title">
-          {{await alert.title}}
-        </p>
-        <div class="vl-alert__message">
-          <p>
-            {{await alert.message}}
-          </p>
-        </div>
-      </div>
-      <button class="vl-alert__close" type="button" {{action "close"}}>
-        <i class="vl-vi vl-vi-cross" aria-hidden="true"></i>
-        <span class="vl-u-visually-hidden">
-          {{t "close"}}
-        </span>
-      </button>
-    </div>
+    <WebComponents::VlAlert
+      class="vl-alert-relative vl-alert-relative--center {{await type}}"
+      aria-live="assertive"
+      @title={{await alert.title}}
+      @message={{await alert.message}}
+      @closeable={{true}}
+      @onClose={{action "close"}}
+    />
   {{/if}}
-  {{#if (gt (await messages.length) 0)}}
-    <div class="toasts-container">
-      {{#each messages as |errorMessage|}}
-        {{utils/custom-messages-toasts
-          errorMessage=errorMessage
-          closeErrorMessage=(action "closeErrorMessage")
-        }}
-      {{/each}}
-    </div>
+  {{#if this.toaster.toasts.length}}
+    {{!-- TODO: Currently, many toasts are automatically generated by the application adapter.
+      A lot of this data-handling doesn't concern the user. We should manually implement toasts on save-
+      operations where it is relevant for the user, and do away with the generated ones.
```

Comment:
Dat vind ik er ook van. Misschien moeten we hiervan een story'ke maken om die rework te doen.
Ik ben daar ook al tegen gebotst op andere plaatsen waar dat niet thuishoorde.

Reply:
Story aangemaakt. Zie [KAS-1421](https://kanselarij.atlassian.net/browse/KAS-1412)